### ISSUE TITLE
Add a label to /search text input

### ISSUE THREAD

**Issue by erinchambers** [CDRH/orchid#291]
Per WCAG 2.0 form inputs should be associated with a label element ([techniques here](https://www.w3.org/WAI/WCAG21/Techniques/html/H44.html))

I've been adding this to individual sites:
  `<label for="q">Search by:</label>`
...but it should probably be in Orchid itself. If we want to visually hide it on Bootstrap sites there's an `.sr-only` class.
